subjects/devops/kept-promise/README.md

```diff
@@ -0,0 +1,52 @@
+## kept-promise
+
+### Instructions
+
+Thing in your code does not always happen as fast as you would like.
+
+Sometimes it is important to wait for a function to finish before keep going on.
+
+Create an asynchronous function `processInfo` in a file `kept-promise.js`. The function `processInfo` will take an asynchronous function as an input, and it will print the message `Ok!` if the return of the input function is zero or a multiple of two, `Error!` otherwise.
+
+The following function can be used as an example of input function for `processInfo`:
+
+```js
+const getImportantInfo = _ =>
+  new Promise(resolve => {
+    setTimeout(() => resolve(Math.round(Math.random() * 10)), 1000)
+  })
+```
+
+> Assume that your function will always get a valid input function 
+
+### Example
+
+The following script `main.js` can be used to test your function:
+
+```js
+import processInfo from './kept-promise.js'
+
+const getImportantInfo = _ =>
+  new Promise(resolve => {
```

Comment:
maybe we could add a verbose hint on how promises work? In a similar way as in the scripting piscine?

Reply:
Adding a `hint` on how to wait for a `Promise` would be almost like showing the solution, that's why I haven't included anything

Reply:
What I mean is a verbose description of what is a promise, and what parameters it takes, as in the exercises in the scripting piscine, for instance:
https://github.com/01-edu/public/tree/master/subjects/devops/burial